python/src/server.py

```diff
@@ -38,10 +40,9 @@ def __init__(self):
 
         Maybe load weights, connect to a database, etc ..
         For example, the following code will load a model saved as a model.pickle file in the models/ directory
-        
         import pickle
-        from konan_sdk.konan_service import constants as Konan_Service_Constants
-        self.loaded_model = pickle.load(open(f"{Konan_Service_Constants.MODELS_DIR}/model.pickle", 'rb'))
+        from konan_sdk.konan_service import constants as Konan_Constants
+        self.loaded_model = pickle.load(open(f"{Konan_Constants.MODELS_DIR}/model.pickle", 'rb'))
         """
         super().__init__()
         # TODO: [3] Initialize your model
```

Comment:
Just wondering, should we create a `load_model` method here that they can override instead of requiring to override init function? it would be simpler for them I think instead of having to call super and will allow us to change the __init__ signature however we want later

Reply:
Would rather keep it in the `__init__()` method because a general `Model` class might be initialized by

- "loading" a file
- "connecting" to a database
- "reading" a value from from an external REST API
- etc ...

Reply:
I mean we could name it `init_model` haha but doesn't matter much for now